integration_test/sendL2msg.test.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-env node */
+// import { instantiateBridge } from './instantiate_bridge'
+;('use strict')
+
+import { BigNumber, ethers, providers, Wallet } from 'ethers'
+import { InboxTools } from '../src/lib/inbox/inbox'
+import { getL2Network } from '../src/lib/dataEntities/networks'
+import { testSetup } from '../scripts/testSetup'
+import { greeter } from './helper/greeter'
+import { expect } from 'chai'
+import { isBytes } from '@ethersproject/bytes'
+
+const sendSignedTx = async (contractCreation: boolean, info?: any) => {
+  const { l1Deployer, l2Deployer } = await testSetup()
```

Comment:
Better to pass these in rather than call `testSetup` again. Should probably only call `testSetup` once per test as it creates some random addresses.